src/Microsoft.VisualBasic/tests/VB/Microsoft.VisualBasic.VB.Tests.vbproj

```diff
@@ -1,9 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{da95454e-5f0e-4f8a-871a-25d1ab62867a}</ProjectGuid>
-    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <Configurations>netstandard-Unix-Debug;netstandard-Unix-Release;netstandard-Windows_NT-Debug;netstandard-Windows_NT-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="SpecialDirectoriesTests.vb" />
     <Compile Include="Dummy.vb" />
+    <Compile Include="FileIOTests.vb" />
+    <Compile Include="TextFieldParserTests.vb" />
```

Comment:
Should these test files be in tests/Microsoft/VisualBasic/... and referenced in Microsoft.VisualBasic.Tests.csproj?

Reply:
tests/Microsoft/VisualBasic/ are C# tests, Dan created this structure starting with Dummy.vb. I don't like that they are in src\Microsoft.VisualBasic\tests\VB because it causes strange behavior with VS where files show up twice. Because each developer created their own tests the organization of all of the tests is inconsistent and somewhat confusing with multiple test directories with identical names, the purpose of the VB directory is to allow tests to be written in VB, I think I am the only one doing VB coded tests now. I tried moving the VB directory to src\Microsoft.VisualBasic\tests.VB and it broke my build and I think it went against the CoreFx structure guidelines where all tests are under Tests and  src\Microsoft.VisualBasic has only ref, src and tests.